Resources/Meta Analysis/description.json

```diff
@@ -1,13 +1,13 @@
 {
 	"moduleDescription" :
 	{
-		"name": 		"Meta Analysis",
```

Comment:
Then the folder name also needs to be changed right?

![Schermafbeelding 2020-06-17 om 09 42 47](https://user-images.githubusercontent.com/18737241/84870625-d4751700-b07f-11ea-89c3-35207cd8d4e2.png)

Reply:
Yes, they should, wait, Ill fix this either by reverting or renaming